lib/modules/versioning/aws-eks-addon/index.ts

```diff
@@ -0,0 +1,22 @@
+import { RegExpVersioningApi } from '../regex';
+import type { VersioningApi } from '../types';
+
+export const id = 'aws-eks-addon';
+export const displayName = 'aws-eks-addon';
+
+export const urls = [];
+
+export const supportsRanges = false;
+
+export class AwsEKSAddonVersioningApi extends RegExpVersioningApi {
+  static versionRegex =
+    '^v?(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)(?<prerelease>-eksbuild\\.\\d+)$';
```

Comment:
```suggestion
    '^v?(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)(?<compatibility>-eksbuild\\.)(?<build>\\d+)$';
```

`prerelease` would force all versions to be unstable, so we use forth version part instead

https://github.com/renovatebot/renovate/blob/68ce0ed2ad7068a7214bf3a5edf81f54701a75e7/lib/modules/versioning/generic.ts#L87-L90